### ISSUE TITLE
CheckOptionGroup Accessibility Voice Over Issue 

### ISSUE THREAD

**Issue by alaneicker1975** [atomikui/atomikui-core#138]
The current aria-descibedby text that is attached to the fieldset does not work. instead, allow for a `ariaDescribedBy` prop to be passed into the `FormOption` component and pass in the associated error and help IDs of the accosiated hint text.

**Comment by alaneicker1975** [comment]
Fixed and merged into master
